### PR TITLE
Automated bilog trimming

### PR DESCRIPTION
**Pull request by vivekanandan-k-rh**
# Description

Automated bilog trimming

Operation:
Create an user
Create a bucket with user credentials
Upload obects to bucket
Verify bucket index log is not empty after creating objects
Wait for default interval of 20 minutes
Verify bucket index log is empty

Passed cephci log: http://pastebin.test.redhat.com/997755

polarion link: https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83572658

# Checklist:

- [ Done] Create a test case in Polarion reviewed and approved.
- [ NA] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ NA] Review the automation design
- [Done ] Implement the test script and perform test runs
- [Done] Submit PR for code review and approve
- [Done] Update Polarion Test with Automation script details and update automation fields
- [NA ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test

### CODE REVIEW

**Review by viduship — APPROVED** [review]

**Review by viduship — COMMENTED** [review]
Hi Vivek,

1. Could you also provide the scenario where trimming happens at the secondary and is verified at primary?

Vidushi

**Comment by rakeshgm** [comment]
@vivekanandan-k-rh can you add this to nautilus too and attach the logs.

File: suites/pacific/rgw/tier1_rgw_multisite_primary_to_secondary.yaml [hunk 1]
```diff
@@ -520,3 +520,15 @@ tests:
...
+      name: Bilog trimming test
+      desc: test bilog trimming
+      polarion-id: CEPH-83572658
+      module: sanity_rgw_multisite.py
+      clusters:
+        ceph-pri:
+          config:
+            script-name: test_bilog_trimming.py
+            config-file-name: test_bilog_trimming.yaml
+            timeout: 300
```
Comment (psathyan) [inline suites/pacific/rgw/tier1_rgw_multisite_primary_to_secondary.yaml:534]: Could you move this test to `tier_1_rgw_extn...`? IMO, let's avoid adding more tests to stage 1 run

**Comment by rakeshgm** [comment]
@vivekanandan-k-rh any update?

**Comment by vivekanandan-k-rh** [comment]
@rakeshgm , Please find the log for nautilus run: http://pastebin.test.redhat.com/998651

@psathyan , Moved the testcase to tier_1_extn for both nautilus and pacific

**Comment by vivekanandan-k-rh** [comment]
> Hi Vivek,
> 
> 1. Could you also provide the scenario where trimming happens at the secondary and is verified at primary?
> 2. Also, it would be good to keep the trimming test cases in the tier_1_extn... yaml since the trimming interval is really long 20 mins. So we can prioritize them as per need .
> 
> Vidushi

@viduship  , Will create separate ticket for bilog trimming from secondary to primary. And I moved to tier_1_extn from tier_1

**Review by rakeshgm — APPROVED** [review]

**Review by psathyan — APPROVED** [review]

**Pull request merged**